locales/en/app.ftl

```diff
@@ -120,6 +120,9 @@ signup-modal-verify-blurb = We sent a verification link to <span id="submitted-e
 signup-modal-verify-expiration = This link expires in 24 hours.
 signup-modal-verify-resend = Not in inbox or spam folder? Resend.
 
+# Appears after Firefox Monitor has sent a verification email to a new user. 
+signup-modal-sent = Sent!
```

Comment:
Where is this being used?
I did <kbd>$ git grep "signup-modal-sent"</kbd> but didn't get any results.

Reply:
So as of this PR the "Sent!" is still being generated by the CSS but I wanted to get the additional strings in asap and at the same time. I'll push something up today that implements the localized string.

Reply:
Ah, perfect!

```sh
$ git grep -n "Sent\!"

public/css/app.css:1115:  content: "Sent!";
```